src/components/HorizontalCard/HorizontalCard.tsx

```diff
@@ -45,6 +47,7 @@ const HorizontalCard = React.forwardRef<HTMLDivElement, HorizontalCardProps>(
       <CardComponent
         ref={ref}
         className={classNames(
+          className,
```

Comment:
I would recommend putting this at the end of these `classNames(...)` args. This way if it's the same specificity as the others class args, it will still override